src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -62,15 +56,7 @@ public void testDoPost_NewUser() throws IOException, ServletException {
     registerServlet.setUserStore(mockUserStore);
 
     registerServlet.doPost(mockRequest, mockResponse);
-
-    ArgumentCaptor<User> userArgumentCaptor = ArgumentCaptor.forClass(User.class);
-
-    Mockito.verify(mockUserStore).addUser(userArgumentCaptor.capture());
-    Assert.assertEquals("test username", userArgumentCaptor.getValue().getName());
-    Assert.assertThat(
-        userArgumentCaptor.getValue().getPasswordHash(), CoreMatchers.containsString("$2a$10$"));
-    Assert.assertEquals(60, userArgumentCaptor.getValue().getPasswordHash().length());
-
+    Mockito.verify(mockUserStore).addUser("test username", "test password", false);
```

Comment:
small comment can go on 'false' here too

Reply:
Thank you, fixed!